src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -8,12 +8,37 @@ namespace Microsoft.Azure.WebJobs.Logging.ApplicationInsights
 {
     public class ApplicationInsightsLoggerOptions
     {
+        /// <summary>
+        /// Gets or sets Application Insights instrumentation key.
+        /// </summary>
         public string InstrumentationKey { get; set; }
 
+        /// <summary>
+        /// Gets or sets sampling settings.
+        /// </summary>
         public SamplingPercentageEstimatorSettings SamplingSettings { get; set; }
 
+        /// <summary>
+        /// Gets or sets snapshot collection options.
+        /// </summary>
         public SnapshotCollectorConfiguration SnapshotConfiguration { get; set; }
 
+        /// <summary>
+        /// Gets or sets authentication key for Quick Pulse (Live Metrics).
+        /// </summary>
         public string QuickPulseAuthenticationApiKey { get; set; }
+
+        /// <summary>
+        /// Gets or sets flag that enables support of W3C distributed tracing protocol
+        /// (and turns on legacy correlation schema).  Enabled by default.
+        /// </summary>
+        public bool EnableW3CDistributedTracing { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets flag that enables injection of multi-component correlation headers into response
+        /// which allows to build Application Map where several ApplicaitonInsights
+        /// instrumentation keys are used.  Disabled by default.
```

Comment:
Can we reprhase this slightly? Assuming my version still makes sense, that is :-)

        /// Gets or sets a flag that enables injection of multi-component correlation headers into responses.
        /// This allows Application Insights to construct an Application Map to  when several
        /// instrumentation keys are used.  Disabled by default.

Reply:
What were the complaints? Performance?

Reply:
[bandwidth](https://github.com/Microsoft/ApplicationInsights-aspnetcore/issues/613) and limited [memory](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/739#issuecomment-367399581) on IoT devices.

Reply:
I think we should just leave it as "true" if that's what the SDK already has. Like you said, most will go through IoT Hub. As long as we document how to turn it off from Functions (via host.json -- we'll own that documentation), it should be fine. And that's more consistent across the SDKs -- and across our settings (both of these new `Enable...` properties default to true).

Reply:
agree, being consistent is good. I just pushed this change